bin/plugin/commands/test/__snapshots__/changelog.js.snap

```diff
@@ -7,22 +7,22 @@ exports[`formatChangelog verify that the changelog is properly formatted 1`] = `
 - Scripts: Webpack configuration update to minimize CSS. ([33676](https://github.com/WordPress/gutenberg/pull/33676))
 
 #### Components
-- components: Use updated range styles. ([33824](https://github.com/WordPress/gutenberg/pull/33824))
-- Components: Update snackbar to use framer motion instead of react spring. ([33717](https://github.com/WordPress/gutenberg/pull/33717))
-- components: Add new ColorPicker. ([33714](https://github.com/WordPress/gutenberg/pull/33714))
-- Components: Promote \`ItemGroup\`. ([33701](https://github.com/WordPress/gutenberg/pull/33701))
+- Add new ColorPicker. ([33714](https://github.com/WordPress/gutenberg/pull/33714))
+- Promote \`ItemGroup\`. ([33701](https://github.com/WordPress/gutenberg/pull/33701))
+- Update snackbar to use framer motion instead of react spring. ([33717](https://github.com/WordPress/gutenberg/pull/33717))
+- Use updated range styles. ([33824](https://github.com/WordPress/gutenberg/pull/33824))
 
 #### Block Library
 - [Post Featured Image]: Add basic dimension controls. ([31634](https://github.com/WordPress/gutenberg/pull/31634))
 
 #### Site Editor
-- Site Editor: Add error boundary. ([33921](https://github.com/WordPress/gutenberg/pull/33921))
+- Add error boundary. ([33921](https://github.com/WordPress/gutenberg/pull/33921))
 
 #### Widgets Editor
 - Add error boundaries to widget screens. ([33771](https://github.com/WordPress/gutenberg/pull/33771))
 
 #### Patterns
-- Block patterns: Add translation context on titles. ([33734](https://github.com/WordPress/gutenberg/pull/33734))
+- Block Add translation context on titles. ([33734](https://github.com/WordPress/gutenberg/pull/33734))
```

Comment:
This one is strange, but I haven't found an easy way to tweak the regular expression.

Reply:
I left a suggestion above to prevent this. It would still output `Block patterns` as a feature in the `Patterns` section unless we start over-optimizing the expression with things like `(Block(s)?\s*)?`, which seems overkill and not worth the extra polishing time it represents.

Reply:
Applied. It works perfectly fine 👍🏻